pwa-devdocs/scripts/create-reference-docs/config/buildpack.js

```diff
@@ -0,0 +1,26 @@
+module.exports = [
+    {
+        target: 'pwa-buildpack/lib/BuildBus/declare-base.js',
+        type: 'function'
+    },
```

Comment:
Does this belong here? The `declare-base.js` file is what declares Buildpack targets, so we wrote the doc blocks in that file. But the targets themselves are in a different category of thing, which should be documented in a separate "Targets" section, right?

Reply:
I can't add another level of nesting to the navigation, so they will all need to go under "Extensibility"